src/components/application_manager/src/rpc_passing_handler.cc

```diff
@@ -228,6 +233,21 @@ void RPCPassingHandler::ForwardRequestToCore(uint32_t correlation_id) {
       std::make_shared<smart_objects::SmartObject>(message);
   rpc_request_queue.erase(correlation_id);
   rpc_request_queue_lock_.Release();
+
+  // Revalidate rpc message before forwarding to core
```

Comment:
Eh, wouldn't say `Revalidate`, it's technically being validated for the first time, just `Validate`.

Reply:
Changed to Validate